### PR TITLE
Zachary King's card

### PR DESCRIPTION
**Pull request by lax0248917**
Added my card information to index.html file, listing 3 stellar developer resources 🚀

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12956,7 +12956,42 @@ <h2>A project for first-time contributions</h2>
...
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://www.udemy.com/course/the-web-developer-bootcamp/" target="_blank"
+                                title="The best web development Udemy course of all time">
+                                Colt Steele's "The Web Developer Bootcamp"
+                            </a>
+                        </li>
+                    </ul>
+                    <!-- COPY everything ABOVE this, from contributor card start to end along with the "START" and "END" comment lines -->
```
Comment (Syknapse) [inline index.html:12990]:
```suggestion
```
You do not need this comment in your card, You should delete it

Comment (Syknapse) [reply index.html]: Ignore this. I just realised we had added the confusing comment to the project. It's been removed now and your PR merged

**Comment by lax0248917** [comment]
Hello Syk,

Was this change already made and committed prior to the merge. Looks like my forked repo and branch is behind in commits, when I pulled in all the changes I see that the comment line you mention has already been taken out.

If you made this change I appreciate it.

Thanks,
_______________________________________________________________________________

Zachary A. King
Developer
Team Armadillo

Phone:  (517) 719-4179
E-mail Address:  zach_a_king@outlook.com

This message and any attachments are the property of Team Armadillo Development and are intended for the named recipients or entity to whom this message is addressed.   If you have received this message in error, please inform the sender via e-mail and destroy the message.  If you are not the intended recipient, you are not allowed to use, copy, or disclose the contents in whole or in part.

**Comment by Syknapse** [comment]
Yes I marked it as resolved because as I was merging your PR and fixing the merge conflicts I removed the comment line. I did this because it wasn't added by you but by an earlier change that I've removed.

**Pull request merged**